Makefile.stubdom

```diff
@@ -280,18 +280,17 @@ build/qemu/config.status: build/qemu/.patched $(FULL_BUILDDEPS)
 		--disable-parallels \
 		--disable-vvfat \
 		--disable-qed \
-		--disable-sheepdog \
 		--disable-auth-pam \
 		--enable-rng-none \
 		${QEMU_OPTIONS} \
 		--cxx=/non-existent \
 		--extra-cflags="-DXEN_PT_LOGGING_ENABLED=1 ${QEMU_EXTRA_CFLAGS}" \
 		--extra-ldflags="${QEMU_EXTRA_LDFLAGS}" \
-		--prefix= 
+		--prefix=/usr
 
-build/qemu/i386-softmmu/qemu-system-i386: build/qemu/.patched build/qemu/config.status build/qemu/.gui-agent-copied-in
-	$(MAKE) -C build/qemu/roms seavgabios-{cirrus,stdvga}
-	$(MAKE) -C build/qemu/pc-bios/optionrom linuxboot.bin
+build/qemu/i386-softmmu/qemu-system-i386: build/qemu/.patched build/qemu/config.status
+	#$(MAKE) -C build/qemu/roms seavgabios-{cirrus,stdvga}
```

Comment:
This line should work as is (at least it works in qemu git checkout), then the resulting binary is copied to `pc-bios` dir.